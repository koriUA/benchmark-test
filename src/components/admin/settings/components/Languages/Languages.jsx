import React from 'react';

const Languages = () => <div>Here goes content for Languages</div>;

export default Languages;
