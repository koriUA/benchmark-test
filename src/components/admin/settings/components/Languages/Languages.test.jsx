import React from 'react';
import { mount } from 'enzyme';

import Languages from './Languages';

describe('Languages', () => {
  test('should render the Languages', () => {
    const component = mount(<Languages />);
    expect(component).toMatchSnapshot();
  });
});
