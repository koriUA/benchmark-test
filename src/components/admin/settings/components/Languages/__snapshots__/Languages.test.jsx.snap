// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Languages should render the Languages 1`] = `
<Languages>
  <div>
    Here goes content for Languages
  </div>
</Languages>
`;
